process.md

```diff
@@ -1,162 +1,143 @@
-# Swift Evolution Process
+# Evolving Swift
 
-Swift is a powerful and intuitive programming language that is designed to make writing and maintaining correct programs easier. Swift is growing and evolving, guided by a community-driven process referred to as the Swift evolution process. This document outlines the Swift evolution process and how a feature grows from a rough idea into something that can improve the Swift development experience for millions of programmers.
+[Swift](http://swift.org) is a powerful and intuitive modern programming language. It was designed from the start to minimize potential programming errors, helping you write and maintain correct code. Swift hasn't reached a stopping point. It continues to change and evolve with an active and involved community. 
 
-## Scope
+Anyone can participate in this process. All it takes is a love of the language and a desire to help make it better. You are invited to join in and help leverage the collective ideas and insight of the Swift community to improve the Swift development experience. Your contributions can improve Swift for millions of programmers.
 
-The Swift evolution process covers all changes to the Swift language and the public interface of the Swift standard library, including new language features and APIs (no matter how small), changes to existing language features or APIs, removal of existing features, and so on. Smaller changes, such as bug fixes, optimizations, or diagnostic improvements can be contributed via the normal contribution process; see [Contributing to Swift](https://swift.org/community/#contributing).
+This document explains how a feature grows from rough ideas to an implementation incorporated into the language standard.  
 
-## Goals
+## You've got an idea for Swift. Now what?
 
-The Swift evolution process aims to leverage the collective ideas, insights, and experience of the Swift community to improve the Swift development experience. Its two primary goals are:
+Congratulations on your idea. Before moving further, perform this sanity check to test whether your change is appropriate for this repository:
+
+* Is it an evolutionary change to the Swift language or to the Swift standard library public interface?
+* Is it a new language feature or API? (no matter how small!)
+* Does it change any existing language features and APIs? 
+* Does it propose to remove any existing features from the language?
+
+If so, you're starting at the right place!
+
+Smaller changes such as bug fixes, optimizations, and diagnostic improvements can be contributed via the normal contribution process. See [Contributing to Swift](https://swift.org/community/#contributing) for details.
```

Comment:
You might want to specify what the "normal process" is: filing bugs, submitting pull requests, etc.